rq/worker.py

```diff
@@ -542,7 +546,7 @@ def heartbeat(self, timeout=0, pipeline=None):
         The effective timeout can never be shorter than default_worker_ttl,
         only larger.
         """
-        timeout = max(timeout, self.default_worker_ttl)
+        timeout = timeout or self.default_worker_ttl
```

Comment:
The docstring mentions that `The effective timeout can never be shorter than default_worker_ttl, only larger.` Since this is no longer true, the docstring should be changed.

Reply:
Done.